src/Aquifer.API/Endpoints/Resources/Content/AssignedToOwnCompany/List/Endpoint.cs

```diff
@@ -23,11 +23,11 @@ public override async Task HandleAsync(CancellationToken ct)
         List<ResourceContentStatus> statuses =
         [
             ResourceContentStatus.New,
-            ResourceContentStatus.AquiferizeInProgress,
-            ResourceContentStatus.TranslationNotStarted,
-            ResourceContentStatus.TranslationInProgress,
-            ResourceContentStatus.AquiferizeManagerReview,
-            ResourceContentStatus.TranslationManagerReview
+            ResourceContentStatus.AquiferizeEditorReview,
+            ResourceContentStatus.TranslationAwaitingAiDraft,
```

Comment:
I think this should be `TranslationAiDraftComplete` since the manager won't have any awaiting draft items assigned to them.